### DISCUSSION TITLE
Discussions are now activated!

### DISCUSSION THREAD

**Discussion by Khhs167** [ReCT-Lang/nrc#5] [Announcements]
Go ham, or something.

I genuinely don't know the difference between issues and discussions hehe :)

**Comment by fiza555** [comment]
discussions are always [activated](https://www.fidesrestoration.com/) hhahaha

**Reply by Khhs167** [reply]
Did you accidentally paste a wrong link or is this spam?

**Reply by Khhs167** [reply]
Actually, to be safe, I’ll file it as spam. Reach out to me on e.g discord if I made a mistake hehe
